rpt/BillsReportMonthlyWard.properties

```
@@ -0,0 +1,21 @@
+#Generated by Eclipse Messages Editor (Eclipse Babel)
+
+Billed.Income = Billed / Income
+
+Cash.Collections = Cash Collections
+
+Month = Month
+
+Receivable = Receivable
+
+Report.From = report from
+
+To = To
+
+Periodical.Incomes.Billed.And.Receivables.Report = Periodical Incomes/Billed and Receivables Report
```

```suggestion
Periodical.Incomes.Billed.And.Receivables.Report = Periodic Incomes/Billed and Receivables Report
```
The typical meaning of periodical is: a magazine or newspaper published at regular intervals.

---

Thanks!

---

ok, but I need to change also the key, new commit comming!